index.ts

```diff
@@ -122,6 +122,12 @@ type ic = {
     msg_cycles_available128: () => nat;
     msg_cycles_refunded: () => nat64;
     msg_cycles_refunded128: () => nat;
+    notify_raw: (
+        canister_id: Principal,
+        method: string,
+        args_raw: blob,
+        payment: nat
```

Comment:
Is this payment the same as our `.with_cycles` flow that we use elsewhere? If so, should we not use that instead? If not, what is this payment?

Reply:
It's not the same, it's entirely separate. This is like call_raw, you just call it like a function and pass in a payment if you wish